src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -372,7 +372,11 @@ def load_tf2_weights_in_pytorch_model(pt_model, tf_weights, allow_missing_keys=F
             raise e
 
         # logger.warning(f"Initialize PyTorch weight {pt_weight_name}")
-
+        # Added a check to see whether the array is a scalar
+        # scalar => should first be converted to numpy arrays
+        # eg. bias terms in Tapas model
+        if numpy.isscalar(array):
+            array = numpy.array(array)
```

Comment:
Not sure if we can add this here, cc @LysandreJik

Reply:
Seconding this - it doesn't seem to have broken any tests but I'm a little scared of it anyway.